src/plugins/FileService/components/UploadDropArea.tsx

```diff
@@ -0,0 +1,125 @@
+import { formatFileSize } from '@dimensiondev/kit'
+import { makeStyles, Typography } from '@material-ui/core'
+import classNames from 'classnames'
+import { isNil } from 'lodash-es'
+import { useSnackbar } from 'notistack'
+import React from 'react'
+import { UploadCloud } from 'react-feather'
+import { useDropArea } from 'react-use'
+import { useI18N } from '../../../utils/i18n-next-ui'
+
+const useStyles = makeStyles({
+    label: {
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'flex-end',
+        alignItems: 'center',
+        position: 'relative',
+        border: '1px dashed #C4C4C4',
+        boxSizing: 'border-box',
+        borderRadius: 4,
+        textAlign: 'center',
+        flex: 1,
+        padding: 12,
+        overflow: 'hidden',
+        userSelect: 'none',
```

Comment:
I think it should add to the `<MaskbookPluginWrapper />` instead.

Reply:
what is this for ?

Reply:
the wrapper component of all plugins. if do like that you haven't to add this css in every places.

Reply:
this a component not is outside container

Reply:
outside container is `src/plugins/FileService/MainDialog.tsx`.